src/gt4py/next/ffront/past_to_itir.py

```diff
@@ -141,16 +142,42 @@ def _visit_stencil_call(self, node: past.Call, **kwargs) -> itir.StencilClosure:
 
```

Comment:
not really part of this pr, but maybe let's change the function signature to
```
def _visit_stencil_call(self, node: past.Call, *, out, domain=None, **kwargs) -> itir.StencilClosure:
```
to avoid copying kwargs, to avoid accidentally using `node.kwargs` instead of `node_kwargs`

Reply:
Read again, `out` and `domain` are not coming from `kwargs`, but are part of the `node.kwargs`. (Or I misunderstand you?)